build.properties

```diff
@@ -1,12 +1,12 @@
 #
 #Sun Jan 24 19:17:22 CST 2016
 minecraft_version=1.12.2
-forge_version=14.23.0.2540
+forge_version=14.23.1.2586
```

Comment:
Hello, Is it a good idea to go beyond the recommended version (14.23.1.2555) ?

Thanks

Reply:
i cannot speak on Ray's behalf, but i would say it was done because of some sort of compat problem or because forge guys recommended it (we are in infrequent contact with them)

Reply:
No, this was done since we needed a new Feature: Capabilities on Chunks, which was introduced later on.